src/social-network/defaults/injectCommentBox.tsx

```diff
@@ -5,9 +5,23 @@ import { useValueRef } from '../../utils/hooks/useValueRef'
 import { CommentBox } from '../../components/InjectedComponents/CommentBox'
 import Services from '../../extension/service'
 import { renderInShadowRoot } from '../../utils/jss/renderInShadowRoot'
+import { dispatchCustomEvents, selectElementContents, sleep } from '../../utils/utils'
 
-export function injectCommentBoxDefault(onPasteToCommentBox: (encryptedComment: string, current: PostInfo) => void) {
+export function injectCommentBoxDefault(
+    onPasteToCommentBox: (
+        encryptedComment: string,
+        current: PostInfo,
+    ) => void = async function onPasteToCommentBoxFacebook(encryptedComment, current) {
```

Comment:
Why should `onPasteToCommentBox` have a default value and it called `onPasteToCommentBoxFacebook`? This should be platform-specific

Reply:
fixed in 82e82d5

Reply:
Yeah I see but I think `root.querySelector('[contenteditable]')` (some website may use textarea or input) and `dispatchCustomEvents('paste', encryptedComment)` (not every website are using React and listenen from `document`) is not a good default.

Reply:
Ok but although ```git blame``` is pointing this edit of line to me, this is actually added more earlier and written by you. ......A todo has been added.